programs/bpf_loader/src/lib.rs

```diff
@@ -680,14 +679,7 @@ fn process_loader_upgradeable_instruction(
                 &invoke_context.feature_set,
             )?;
 
-            // Skip writing true to executable meta after bpf program deployment when
-            // `deprecate_executable_meta_update_in_bpf_loader` feature is activated.
-            if !invoke_context
-                .feature_set
-                .is_active(&deprecate_executable_meta_update_in_bpf_loader::id())
-            {
-                program.set_executable(true)?;
-            }
+            program.set_executable(true)?;
```

Comment:
Any reason to revert this?

Reply:
Without this change, test `test_program_sbf_invoke_in_same_tx_as_deployment` fails with:
```
---- test_program_sbf_invoke_in_same_tx_as_deployment stdout ----
thread 'test_program_sbf_invoke_in_same_tx_as_deployment' panicked at tests/programs.rs:1908:13:
assertion `left == right` failed
  left: InstructionError(2, AccountNotExecutable)
 right: InstructionError(2, InvalidAccountData)
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
So the error code changes.

Reply:
That's ok, it is feature gated after all.

Reply:
Good point. I've left this as-is and updated the error in the test instead.